mars/deploy/local/tests/test_cluster.py

```diff
@@ -478,6 +479,11 @@ def testEagerMode(self, *_):
                     expected2 = expected1.dot(expected1)
                     np.testing.assert_array_almost_equal(r2, expected2)
 
+                    web_session = Session.default_or_local()._sess
+                    tasks_url = 'http://{0}/api/session/{1}/graph'.format(cluster.web_endpoint,
```

Comment:
Can we add a web API like `n_tasks` to make it more clean?

Reply:
OK, I wasn't sure it's necessary to add an API for it.